source/scale/push-notification-health-targets.rst

```diff
@@ -0,0 +1,86 @@
+  :start-after: :nosearch:
+
+When using the `Mattermost Notification Health <https://grafana.com/grafana/dashboards/21305-mattermost-notification-health>`_ Grafana dashboard to track different types of notifications sent from Mattermost, we recommend adhering to the following mobile push notification health targets to ensure a performant production deployment of Mattermost.
+
+.. note::
+
+    - Accessing and enabling Mattermost Notification Health Monitoring requires the feature flag ``NotificationMonitoring`` to be set to ``true``. 
+    - System admins can :ref:`disable notification monitoring data collection <configure/site-configuration-settings:enable notification monitoring>` through the System Console.
+
+Push Proxy Delivery Rate
+------------------------
+
+The **Push Proxy Delivery Rate** panel indicates the percentage of push notifications that have been successfully delivered to the endpoints (i.e., mobile devices) out of the total number of push notifications sent.
+
+Monitor this delivery rate to ensure that users are receiving their notifications in a timely manner. This value should always be 100%. Anything below 99% should be investigated further to identify issues with the push notification configuration, connectivity problems, or possible service interruptions with the push proxy provider.
+
+.. image:: ../images/push-proxy-delivery-rate.png
+  :alt: The Push Notifications Errors by Reason panel provides admins with a breakdown of the errors encountered when sending push notifications, categorized by the specific reason for each error.
+  :scale: 50
+
+Push Notifications Errors by Reason
+------------------------------------
+
+The **Push Notifications Errors by Reason** panel is a critical tool for maintaining the reliability and effectiveness of the push notification system. These values provide a breakdown of the errors encountered when sending push notifications, categorized by the specific reason for each error. 
+
+This information helps admins to pinpoint common problems affecting push notification delivery, diagnose and fix issues more efficiently, detect patterns and trends, identify intermittent issues before they become more serious, and enhance the overall user experience with the Mattermost platform.
+
+Missing profile errors are generally safe to ignore. However, if you see occurences of this error increase, verify logs for ``push_proxy_send_error``.
+
+Push Notifications Not Sent Distribution
+------------------------------------------
+
+The **Push Notifications Not Sent Distribution** panel helps admins ensure the robustness and reliability of the push notification system by providing detailed insights into the specific reasons for notification delivery failures. This panel displays the distribution of push notifications that were not sent, categorized by the specific reasons why they were not sent. This information helps admins understand the causes behind undelivered push notifications. 
```

Comment:
Not really delivery failures. In this context, it is not a failure the notification was not sent. It is expected that the notification was not sent, and this tell us the reason why that is expected.

For example, a notification is not sent, if the channel is muted.